ccd-definition/ComplexTypes/LengthOfUnemploymentComplexTypeLRspec.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "ID": "LengthOfUnemploymentComplexTypeLRspec",
+    "ListElementCode": "lengthOfUnemploymentLabel",
+    "ElementLabel": "How long has your client been unemployed? \n",
```

Comment:
LengthOfUnem...ComplexType has a Label field, so where it is actually used it doesn't use a label (while it could)
Instead, you can create a complex type YearsMonths with just the years and months fields as below, use that type, use the label field where you use now LengthOfUnem...ComplexType and thus we have a reusable complex type in case this is not the only time they want a years-and-months double field

Reply:
There are no changes for me in this field. I just merge it from the other branch